### DISCUSSION TITLE
Milvus displays 0% Is Loaded

### DISCUSSION THREAD

**Discussion by stan5215** [milvus-io/milvus#29483] [Q&A and General discussion]
Hi team,
We are trying to ingest 1 billion vectors in milvus with below setup:
**Environment**
> - Milvus version: 2.3.3
> - Deployment mode(standalone or cluster): Milvus Standalone with Docker Compose (CPU)
> - OS(Ubuntu or CentOS): CentOS
> - CPU/Memory: 8/8GB
> - GPU: no
> num_shards of collection: 3 
> 1 collection with 4 partitions
> Vector dim: 1536
> Scale: 40 thousand

**Current Behavior**
After upgrading to v2.3.x, I have been using it normally for two months. But after restarting the server, it suddenly prompts 0% Is Loaded, and then it can no longer run normally.

It happened once a month ago. I restored it through backup at that time, but now the backup also failed.

**The most related GitHub Discussions link**
[28138](https://github.com/milvus-io/milvus/issues/28138)
[26754](https://github.com/milvus-io/milvus/issues/26754)

I tried the solution in issues, but it didn't work.

How do I fix it?

[backup.log](https://github.com/milvus-io/milvus/files/13769513/backup.log)

[milvus.log](https://github.com/milvus-io/milvus/files/13769515/milvus.log)
[docker-compose.txt](https://github.com/milvus-io/milvus/files/13769524/docker-compose.txt)

**Comment by yhmo** [comment]
This is a complicated issue, link to https://github.com/milvus-io/milvus/issues/29484
